src/repos/UserRepo.ts

```diff
@@ -20,13 +23,22 @@ const createUser = async (
     const followedPublications = followedPublicationsIDs.map((id) => {
       return Object.assign(new PublicationID(), { id });
     });
-
     const uuid = uuidv4();
+
+    const weeklyDebrief = Object.assign(new WeeklyDebrief(), {
+      uuid,
+      createdAt: new Date(),
+      expirationDate: new Date('December 17, 1995 03:24:00'),
```

Comment:
I've always wondered what the significance of this date is

Reply:
+1

Reply:
should just be curr date + 7

Reply:
What behavior should happen if the user is created in the middle of the week?

Reply:
This is the weeklydebrief that is created for new users, so shouldn't it start out as expired?

Reply:
You are correct. I rescind my earlier comment. Instead, make this a constant such as `IMMEDIATELY_EXPIRE_DATE` or something. It's unclear what a hardcoded date is - future developers reading the code may not understand the intention of it. Changing it to a constant with an accurate name will display the intention of setting it to a date that immediately expires the weekly debrief.